crates/anstream/src/adapter/strip.rs

```diff
@@ -174,19 +174,22 @@ fn is_utf8_continuation(b: u8) -> bool {
 /// let plain_str = anstream::adapter::strip_bytes(styled_text.as_bytes()).into_vec();
 /// assert_eq!(plain_str.as_slice(), &b"foo bar"[..]);
 /// ```
+#[cfg(feature = "utf8")]
 #[inline]
 pub fn strip_bytes(data: &[u8]) -> StrippedBytes<'_> {
```

Comment:
Taking built-in functionality and putting it behind a feature is a breaking change as it may break people who use `default-features = false`